Hello-world.js

```diff
@@ -1,3 +1,3 @@
-const == 10;
+const number = 3;
 let word = "Hello World";
-console.lof(word);
\ No newline at end of file
+console.log(word);
```

Comment:
- [ ] I will encourage you to follow the project requirements by removing the `Hello-world.js` file. We really don't need it now